exercises/concept/ellens-alien-game/.meta/design.md

```diff
@@ -0,0 +1,44 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to introduce the student to the concept of classes.
+
+## Learning objectives
+
+- Understand the _basic_ idea behind Object Oriented Programming (OOP).
+- Learn what a class represents.
+- Learn what an object is.
+- Understand the difference between classes and objects.
+- Know how to create a class.
+- Know how to create objects.
+- Understand that initializing a class creates an object.
+- Know about 'constructors' and how they are used to initialize the object upon instantiation.
+- Know that `constructors` are called upon initialization/object creation.
```

Comment:
either \`constructors\` (with backticks) or 'constructors' (with singe quotes) but not both